webapp/src/main/java/org/opentestsystem/rdw/admin/repository/impl/JdbcGroupRepository.java

```diff
@@ -57,8 +61,24 @@
         );
     }
 
+    // TODO: This implementation only updates the isDeleted field.
+    // TODO: Other fields need to be validated before they can be updated.
+    @Override
+    public void update(@NotNull final PermissionScope permissionScope, @NotNull final Group group) {
```

Comment:
Right now it seems the UI is setting the deleted flag and doing a normal update.  The soft-delete feels like an implementation detail where the front-end should just say it wants to delete it and the service deletes it.  It feels to me like these update methods should be delete and the API should use a DELETE verb, and then we just happen to be implementing that delete as a soft-delete by updating a specific flag in the DB.